en/activity-standard/iati-activities/iati-activity/tag/description.rst

```diff
@@ -0,0 +1,16 @@
+Example Usage
```

Comment:
`iati-activities/iati-activity/tag/description` is not included in the [2.03 Upgrade - Final Technical Summary](https://docs.google.com/spreadsheets/d/1XWnzesMvLebIVLk47OIXATfhcjESVcaGnw8SJpOuCCk/edit#gid=595223309) (table lines 157-163) as implmentated in IATI/IATI-Schemas#342.

Will remove this page and references in `en/activity-standard/iati-activities/iati-activity/tag/description/narrative.rst`.

Reply:
Fixed in e7ef28b